File: deaf_grandma.py
import re
def grandma():
    count = 0
    granny = input("HEY KID!")
    while count < 2:
        response = input()
        if response == "":
            print("what!")
        elif re.search("[a-z]", response):
            print("SPEAK UP, KID!")
        elif response == response.upper() and not response == "GOODBYE!":
            print("NO, NOT SINCE 1946!")
        elif response == "GOODBYE!":
            count += 1
            print("LEAVING SO SOON?")
    return("LATER, SKATER!")
print(grandma())

File: deaf_grandma.js
const prompt = require("prompt-sync")({ sigint: true });
let goodbye = 0;
let greeting = prompt("HEY KID!");
while (goodbye < 2) {
  response = prompt();
  if (response === "") {
    console.log("what!");
  } else if (response.match(/[a-z]/g)) {
    console.log("SPEAK UP, KID!")
  } else if (response === response.toUpperCase() && response !== "" && response !== "GOODBYE!") {
    console.log(`NO, NOT SINCE 1946!`)
  } else if (response === "GOODBYE!") {
    console.log("LEAVING SO SOON?")
    goodbye ++
  }
  if(goodbye === 2){
    console.log("LATER, SKATER!")
  }
}
